.gitignore

```diff
@@ -0,0 +1 @@
+./public_html/wp-content/mu-plugins/blocks/assets/blocks.min*
```

Comment:
Done

Reply:
The new commit message is a good improvement, but it still doesn't include the reason commit is being made in the short description. That's equally (if not more) important the describing _what_ the commit is doing (since that is often obvious from the diff). 

If we don't say _why_ a change is made, our future selves may be confused when trying to understand the reason, which can impact our ability to make informed decisions about future changes.